examples/jsm/controls/TrackballControls.js

```diff
@@ -12,7 +12,7 @@ const _endEvent = { type: 'end' };
 
 class TrackballControls extends EventDispatcher {
 
-	constructor( object, domElement ) {
+	constructor( object, domElement, multiTouchRotate = false ) {
```

Comment:
Sorry for the late feedback. Do you mind removing `multiTouchRotate` as a ctor parameter and just add it as a property (with `false` as a default value)? In this way, the ctor keeps its consistency compared to other contorls.

Reply:
done